### DISCUSSION TITLE
Plugin [filament-fullcalendar] is not registered for panel [admin].

### DISCUSSION THREAD

**Discussion by DevNatthapong** [saade/filament-fullcalendar#153] [Q&A]
`  ->plugin(
                ThemesPlugin::make(),
                FilamentFullCalendarPlugin::make()
                    ->selectable()
                    ->editable()
                    ->timezone('Asia/Bangkok')
                    ->locale('th')
                    ->plugins(['dayGrid', 'timeGrid'])
            )`

**Comment by sasori13L** [comment]
->plugins([ThemesPlugin::make(), FilamentFullCalendarPlugin::make()]); 

change the plugin to plugins and make it an array value

https://laraveldaily.com/post/filament-show-calendar-of-tasks-with-fullcalendar
